helix/views.py

```diff
@@ -336,6 +337,39 @@ def helix_reso_export_xml(request):
 
     return HttpResponse(rendered_xml, content_type='text/xml')
 
+@api_endpoint
+@api_view(['GET'])
+def helix_solar_production(request, pk=None):
+    org_id = request.GET['organization_id']
+    user = request.user
+    property_state = PropertyState.objects.get(pk=pk)
+    lat = None
+    lng = None
+    capacity = 0
+    errors = []
+    if 'Lat' in property_state.extra_data:
+        lat = property_state.extra_data['Lat']
+    else:
+        errors.append('Property has no Lat column defined')
+    if 'Long' in property_state.extra_data:
+        lng = property_state.extra_data['Long']
+    else:
+        errors.append('Property has no Long column defined')
+    if 'CAP Electric PV' in property_state.extra_data:
+        capacity = property_state.extra_data['CAP Electric PV']
+        capacity = simplejson.loads(capacity)['quantity']
+    else:
+        errors.append('Property has no CAP Electric PV column defined')
```

Comment:
We'lll want to fetch the capacity from propertyview > greenassessmentproperty > measurement where measurement_type = CAP.  There could be more than one in which case we'll want to iterate through them

Reply:
How do I get from PropertyState to PropertyView? Using the following (as done in [helix/views.py#L348](https://github.com/ClearlyEnergy/HELIX/blob/master/helix/views.py#L348)) throws an error:
```
property_view = PropertyView.objects.get(state=property_state)
seed.models.properties.DoesNotExist: PropertyView matching query does not exist.
```